mlflow/server/auth/__init__.py

```diff
@@ -824,72 +816,72 @@ def _enable_auth(app: Flask):
     create_admin_user(auth_config.admin_username, auth_config.admin_password)
 
     app.add_url_rule(
-        rule=ROUTES.SIGNUP,
+        rule=SIGNUP,
         view_func=signup,
         methods=["GET"],
     )
     app.add_url_rule(
-        rule=ROUTES.CREATE_USER,
+        rule=CREATE_USER,
         view_func=create_user,
         methods=["POST"],
     )
     app.add_url_rule(
-        rule=ROUTES.GET_USER,
+        rule=GET_USER,
         view_func=get_user,
-        methods=["POST"],
+        methods=["GET"],
```

Comment:
using GET here should be more appropriate

Reply:
makes a perfect sense